Dockerfile

```diff
@@ -0,0 +1,109 @@
+# ===============================================
+FROM helsinkitest/node:12-slim as appbase
+# ===============================================
+
+# Offical image has npm log verbosity as info. More info - https://github.com/nodejs/docker-node#verbosity
+ENV NPM_CONFIG_LOGLEVEL warn
+
+# Global npm deps in a non-root user directory
+ENV NPM_CONFIG_PREFIX=/app/.npm-global
+ENV PATH=$PATH:/app/.npm-global/bin
+
+# Set env
+ARG PORT
+ENV PORT $PORT
+
+# Use non-root user
+USER appuser
+
+# Copy package.json and package-lock.json/yarn.lock files
+COPY --chown=appuser:appuser package*.json *yarn* ./
+
+# Copy internals becaused they are called in preinstall and postinstall
+# hooks
+COPY --chown=appuser:appuser internals/ ./internals/
+
+# Install npm depepndencies
```

Comment:
The comment does not correspond to the command

Reply:
This comment is "inherited" from here: https://github.com/City-of-Helsinki/kukkuu-ui/blob/develop/Dockerfile#L26

I had a hard time understanding whether the original comment was just left behind after editing the file or not. I now changed the comment to something more generic:
>Make scripts in dependencies available through path